packages/react-rest/README.md

```diff
@@ -167,12 +178,12 @@ const jsonapi = createClient("/api", {
 const hatchedReactRest = hatchifyReactRest({ Todo }, jsonapi)
 
 function App() {
-  const [todos, listState] = hatchedReactRest.Todo.useAll()
+  const [todos, todosMeta] = hatchedReactRest.Todo.useAll()
```

Comment:
`meta` feels more accurate than `state`. Additionally, I changed `listState` to `todosMeta` so that we can do the same with `usersMeta` later on